cmd/lookout/serve.go

```diff
@@ -91,6 +91,21 @@ func (c *ServeCommand) Execute(args []string) error {
 		return err
 	}
 
+	db, err := c.initDB()
+	if err != nil {
+		return fmt.Errorf("Can't connect to the DB: %s", err)
```

Comment:
Could you check if that should follow https://github.com/golang/go/wiki/CodeReviewComments#error-strings ?

Reply:
I usually follow that convention, but in this case I capitalized it to make it consistent with the previous messages, like "Can't open configuration file".
Do you think I should I change all 3 messages?

Reply:
IDK, up to you - but consistency wins and this also might be top-level error

Reply:
merged, we can change this later